pennylane/templates/subroutines/fermionic_double_excitation.py

```diff
@@ -49,30 +49,30 @@ def _layer1(weight, s, r, q, p, set_cnot_wires):
         q (int): qubit index ``q``
         p (int): qubit index ``p``
         set_cnot_wires (list[Wires]): list of CNOT wires
-    """
 
+    Returns:
+          list[.Operator]: sequence of operators defined by this function
```

Comment:
So now these auxiliary functions create their own `op_list` and return it, so the final op_list can be extended with the ops collected here.

Reply:
much nicer!